src/forms/formHelpers.js

```diff
@@ -23,29 +22,6 @@ import { toMoneyStr } from '../utils/prettifiers';
 // GENERIC COMPONENTS
 // ========================================
 
-/** Returns a Grid Column containing a button of the style used
-* to navigate backwards and forwards through steps of the form.
-*
-* @function
-* @param {object} props
-* @property {object} props.func - A function that is run when the
-* button is clicked.
-* @property {string} props.name - The text to be displayed on
-* the button.
-*
-* @returns Component
-*/
-const BottomButton = function(props){
```

Comment:
This was here because of the 'Save' button that used to exist. I'm not sure if we're now going to replace it with a 'Finish'/'Erase data' button, so it may be worth keeping the button separate, or abstracting (as I started to do in a different PR, though if we do that, it'd probably be worth doing almost what you did here).

Reply:
Hmm. I got rid of the `<Grid>` so there was nothing in `<BottomButton>` other than a `<Button>` with a few extra props. But if its used elsewhere...